compile_commands.json

```diff
@@ -0,0 +1,92 @@
+[
+ {
+  "directory": "/home/sgj/codes/fy/my_toy_compiler",
```

Comment:
This should be updated to avoid hard coding paths or this entire file should be removed since it's not related to the changes.

Reply:
my bad. I'll be careful next time.